{"name": "test_multiple_login_page_success[invalid_user1-wrong_password]", "status": "passed", "parameters": [{"name": "username", "value": "'invalid_user1'"}, {"name": "password", "value": "'wrong_password'"}], "start": 1753501930744, "stop": 1753501931578, "uuid": "40010d7d-5498-4b4f-9986-3dfa667b14e9", "historyId": "67e9264d245abd41856c5586200953c3", "testCaseId": "90d4f72dd78db4ae114ce1f1cc9587a0", "fullName": "tests.test_multiple_login#test_multiple_login_page_success", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_multiple_login"}, {"name": "host", "value": "ahmads-MacBook-Pro.local"}, {"name": "thread", "value": "51691-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_multiple_login"}], "titlePath": ["tests", "test_multiple_login.py"]}